api.go

```diff
@@ -1,65 +1,48 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"os"
-
```

Comment:
let me know if this newline is a convention I'm unaware of!

Reply:
when I write go in vs code it automatically generates a new line. So I guess it is a convention